PWGJE/EMCALJetTasks/UserTasks/AliAnalysisTaskRawJetWithEP.h

```diff
@@ -1,88 +1,49 @@
-
-#ifndef AliAnalysisTaskRawJetWithEP_H
-#define AliAnalysisTaskRawJetWithEP_H
-/**
- * \file AliAnalysisTaskRawJetWithEP.h
- * \brief Declaration of class AliAnalysisTaskRawJetWithEP
- *
- * In this header file the class AliAnalysisTaskRawJetWithEP is declared.
- * This is a sample task that shows how to write a simple user analysis task
- * using the EMCal jet framework. It is also used to do automatic benchmark
- * tests of the software.
- *
- * \author Salvatore Aiola <salvatore.aiola@cern.ch>, Yale University
- * \date Apr 27, 2016
- */
-
-/* Copyright(c) 1998-2016, ALICE Experiment at CERN, All rights reserved. *
- * See cxx source for full Copyright notice                               */
-
+#ifndef ALIANALYSISTASKEMCALJETSAMPLE_H
+#define ALIANALYSISTASKEMCALJETSAMPLE_H
```

Comment:
This will interfere with the sample task, and may lead to unexpected behavior. Please return these two lines to what you had before:

```c++
#ifndef ALIANALYSISTASKRAWJETWITHEP_H
#define ALIANALYSISTASKRAWJETWITHEP_H
```

(should have been all caps by convention)

Reply:
Thank you notice me that. I forgot to fix these lines. Now I modified them.